examples/minimal-expo/metro.config.js

```diff
@@ -0,0 +1,47 @@
+const { getDefaultConfig } = require("expo/metro-config");
+const { FileStore } = require("metro-cache");
+const path = require("path");
+
+module.exports = withTurborepoManagedCache(
+  withMonorepoPaths(getDefaultConfig(__dirname)),
+);
+
+/**
+ * Add the monorepo paths to the Metro config.
+ * This allows Metro to resolve modules from the monorepo.
+ *
+ * @see https://docs.expo.dev/guides/monorepos/#modify-the-metro-config
+ * @param {import('expo/metro-config').MetroConfig} config
+ * @returns {import('expo/metro-config').MetroConfig}
+ */
+function withMonorepoPaths(config) {
+  const projectRoot = __dirname;
+  const workspaceRoot = path.resolve(projectRoot, "../..");
+
+  // #1 - Watch all files in the monorepo
+  config.watchFolders = [workspaceRoot];
+
+  // #2 - Resolve modules within the project's `node_modules` first, then all monorepo modules
+  config.resolver.nodeModulesPaths = [
+    path.resolve(projectRoot, "node_modules"),
+    path.resolve(workspaceRoot, "node_modules"),
+  ];
```

Comment:
expo/metro-config does this automatically.

Reply:
it's from the docs: https://docs.expo.dev/guides/monorepos/#modify-the-metro-config

Reply:
Setting `nodeModulesPaths` shouldn't be necessary at all afaik?